packages/editor/src/components/post-title/index.native.js

```diff
@@ -39,10 +33,7 @@ class PostTitle extends Component {
 	}
 
 	focus() {
-		if ( this.titleViewRef ) {
-			this.titleViewRef.focus();
-			this.props.onSelect();
-		}
+		this.props.onSelect();
```

Comment:
Because the selected state is [being passed down as the `RichText` component's `__unstableIsSelected` prop](https://github.com/WordPress/gutenberg/blob/c68e2da4aec1b58f91ff048c65873189374fbe14/packages/editor/src/components/post-title/index.native.js#L86), we no longer need to keep a ref and manually call focus. When the `__unstableIsSelected` prop is updated to true, the `RichText` component [takes care of focusing itself](https://github.com/WordPress/gutenberg/blob/master/packages%2Frich-text%2Fsrc%2Fcomponent%2Findex.native.js#L734).